v2.0/update.md

```diff
@@ -16,23 +16,26 @@ The user must have the `SELECT` and `UPDATE` [privileges](privileges.html) on th
 
 ## Synopsis
 
-<div>{% include sql/v2.0/diagrams/update.html %}</div>
+{% include sql/{{ page.version.version }}/diagrams/update.html %}
+
+<div markdown="1"></div>
 
 ## Parameters
 
 Parameter | Description
 ----------|------------
+`common_table_expr` | Docs coming soon.
 `table_name` | The name of the table that contains the rows you want to update.
-`AS name` | An alias for the table name. When an alias is provided, it completely hides the actual table name.
+`AS table_alias_name` | An alias for the table name. When an alias is provided, it completely hides the actual table name.
 `column_name` | The name of the column whose values you want to update.
-`a_expr` | The new value you want to use, the [aggregate function](functions-and-operators.html#aggregate-functions) you want to perform, or the [value expression](sql-expressions.html) you want to use.
+`a_expr` | The new value you want to use, the [aggregate function](functions-and-operators.html#aggregate-functions) you want to perform, or the [scalar expression](scalar-expressions.html) you want to use.
 `DEFAULT` | To fill columns with their [default values](default-value.html), use `DEFAULT VALUES` in place of `a_expr`. To fill a specific column with its default value, leave the value out of the `a_expr` or use `DEFAULT` at the appropriate position.
 `column_name_list` | A comma-separated list of column names, in parentheses.
```

Comment:
We inlined this in the diagram, so there's a mismatch. We could either not inline in the diagram, or update this row.